package.json

```diff
@@ -62,12 +63,15 @@
     "eslint-plugin-node": "^9.1.0",
     "eslint-plugin-promise": "^4.1.1",
     "eslint-plugin-react": "^7.14.2",
-    "eslint-plugin-standard": "^4.0.0",
+    "eslint-plugin-standard": "^4.0.0",    
     "file-loader": "^3.0.0",
     "html-webpack-plugin": "^3.2.0",
     "jest": "^24.8.0",
+    "node-fetch": "^2.6.0",
     "nodemon": "^1.17.3",
+    "react-test-renderer": "^16.8.6",
```

Comment:
Why do you need this library?
And same for node-fetch

Reply:
with node-fetch trying to make mock post, deleted now